libr/core/cmd_type.c

```diff
@@ -267,8 +267,22 @@ static void save_parsed_type(RCore *core, const char *parsed) {
 	}
 }
 
+static RCore *core_ = NULL; // HACK
```

Comment:
globals are evil

Reply:
I don't like this part too.

Reply:
I don't like it either but doing away with the global apparently requires significant work in sdb -- there should be a way to pass a context to sdb_foreach_list_filter().

Reply:
That should be in a different PR imho.